Week2/js-exercises/myFavoriteHobbies/favoriteHobbiesIndex.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>My Favorite Hobbies</title>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+
+</head>
+
+<body>
+
+
+    <h1 id="header">My favorite hobbies are:</h1>
+
+    <script
+        src="C:\Users\Nikos Sp\Documents\GitHub\JavaScript2\Week2\js-exercises\myFavoriteHobbies\myFavoriteHobbies.js"></script>
```

Comment:
This is an absolute import. If I try to test in my machine this file it will not work. Try using relative paths
https://phpdelusions.net/articles/paths